lib/mountain_view/example.yml

```diff
@@ -0,0 +1,10 @@
+
+- :meta: 'information about the component' #optional
+  :stubs: #as many components as you need
+    -
+      :id: 1
+      :title: "iyevuycveuyv"
+      :subtitle: "ebcecbeuc"
+    -
+      :id: 2
+      :title: "Yejvceuuvcue"
```

Comment:
Is there a reason you made this an array? (With the `-` at the beginning)

If it's just a hash, we can save a lot of `.first` calls on the `styleguide_stubs` method in the `MountainView::Component` class. But I may be missing something.